dts/arm/nxp/MKE18F512VLH16-pinctrl.dtsi

```diff
@@ -0,0 +1,10779 @@
+/*
+ * NOTE: Autogenerated file
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&porta {
+	adc0_se_pta7: adc0_se_pta7 {
+		nxp,kinetis-port-pins = < 7 0 >;
+	};
```

Comment:
Just picking a random pin/function for this comment: Should the function here be `adc0_se2_pta7`? Otherwise, how would you know which ADC0 single-ended input, you are mapping?

Reply:
Will fix, if you notice any others please let me know.

Reply:
From a quick glance, this comments seems to affect all "numbered" peripheral pins (e.g. `cmp1_in_pta7` => `cmp1_in1_pta7`, `ftm0_flt_pta7` => `ftm0_flt2_pta7`, ...).

Reply:
This should now be fixed up.